OpenRA.Game/Graphics/Viewport.cs

```diff
@@ -74,6 +81,23 @@ public float Zoom
 		public static int TicksSinceLastMove = 0;
 		public static int2 LastMousePos;
 
+		float ClosestTo(IEnumerable<float> collection, float target)
+		{
+			float closest_value = collection.First();
```

Comment:
OpenRA code style is to use `var` instead of explicit types for local variables.